### PR TITLE
Enable babel targets

### PR DESCRIPTION
**Pull request by hkollmann**
With this change it's possible to influence the babel process.
 For example you can add 

   "babelOptions": {
      "targets": {
         "edge": "13"
      }
   },

and modern syntax will not be translated anymore.

### CODE REVIEW

File: docs/configuration/compile.md [hunk 1]
```diff
@@ -94,8 +94,8 @@ The `targets` key is an array of objects, one for each possible target that can
 - `typescript` - see below
 - `minify` - (**optional**) determines the minification to be used for applications, if the target supports it; can be overridden on a per application basis.  Can be `off`, `minify`, `mangle`, or `beautify`.
 - `addCreatedAt` - (**optional**) if true, this will cause every object to have a hidden property called `$$createdAt` which points to an object containing `filename`, `lineNumber`, and `column` properties
-
-If you want to use more than the three default target types and/or use custom target classes, you can use the `targetClass` key to supply the name of the class as a string.
```
Comment (cboulanger) [inline docs/configuration/compile.md]: Did you remove this piece of documentation on purpose? Since it seems unrelated to your PR.

Comment (hkollmann) [reply docs/configuration/compile.md]: no it's an accident..

**Comment by johnspackman** [comment]
merging, thank you @hkollmann
